### PR TITLE
SCRUM-1709

### PR DESCRIPTION
**Pull request by kthorat-prog**
Pushing branch now to check save api (as Literature references need a vpn connection on local system)

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java [hunk 1]
```diff
@@ -15,6 +17,15 @@
 
 import lombok.*;
 
+@JsonTypeInfo(
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java:20]: @kthorat-prog I don't think we want this at both levels?

Comment (kthorat-prog) [reply src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java:20]: So shall I remove it from genomic Entity?

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java:20]: Was it solving an issue? I think it depends on what the subject field in the disease annotation is pointing to.

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java:20]: So Gene, Allele, and AGM are all sub classes of GenomicEntity so I would guess we would want it at that level.

Comment (kthorat-prog) [reply src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java:20]: Yes. I think so too.

Comment (kthorat-prog) [reply src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java:20]: I have removed type from BiologicalEntity class .Please check.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**